modules/modules.nix

```
@@ -23,6 +23,7 @@ let
     (loadModule ./home-environment.nix { })
     (loadModule ./manual.nix { })
     (loadModule ./misc/dconf.nix { })
+    (loadModule ./misc/pulseaudio.nix { })
```

I assume this module wouldn't make sense on macOS?
```suggestion
    (loadModule ./misc/pulseaudio.nix { condition = hostPlatform.isLinux; })
```